docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+    api-dummy-fpm:
+        build:
+            context: ./api-dummy
+        volumes:
+            - ./api-dummy:/srv/app
```

Comment:
this is a source-based workflow, which is enough for `dev` and `ci` environments (possibly also `demo`). Shouldn't be a problem to extend it later to allow deployment without the source code.